V3.0 - @KyleRosenberg changes

KyleRosenberg opened the pull request:
For some reason there are duplicates in the splitText array with different values, so I added checks for the two that were giving me problems. Not sure if any others are bad.

Commit:
Fix info()

For some reason major and minor have two indexes, one of which is
empty, so I added a check to make sure each was added (my major wasn’t
before).
Also changed the try catch in the example so raw_input can be used
anywhere.

Commit:
Removed print

biw commented on myCUinfo.py:
```diff
@@ -173,13 +171,20 @@ def info(self):
-        else:
+        info["minor"] = splitText[12 + employeeShift].strip()[16:-17]
+        if info["minor"] == "":
+            info["minor"] = splitText[18 + employeeShift].strip()[16:-17]
```
I don't have a major on MyCUinfo and I get `IndexError: list index out of range` on this call. I know the rest of the code also needs this, but would you mind adding some error catching for this case

KyleRosenberg replied:
I might just change it to iterate the splitText array for data since idk if those values can be hard coded anymore. I will change it after I fix classes()

biw replied:
Sounds good

Commit:
Fix classes()

I ran into trouble with a missing instructor for a recitation and I
have not added checks for other empty fields which may or may not be a
problem.

Commit:
Fix info() again

Commit:
Fixed books()

Some of the old data is no longer available
Also changed example to a class with multiple books for testing purposes

KyleRosenberg commented:
I'm relatively new to github so I'm not sure what to do about the changes requested but I have committed fixes for all the cuSession functions

biw reviewed (approved):
Looks great!
